Source/santad/SNTDaemonControlController.m

```diff
@@ -242,6 +243,18 @@ - (void)setEnableTransitiveRules:(BOOL)enabled reply:(void (^)(void))reply {
   reply();
 }
 
+#pragma mark Metrics Ops
+
+- (void)metrics:(void (^)(NSDictionary *))reply {
+  // If metrics are not enabled send nil back
+  if (![[SNTConfigurator configurator] exportMetrics]) {
```

Comment:
Maybe also LOGI/LOGE here?

Reply:
I could see LOGD, but if you don't have metrics configured but run the metrics command then this should just return.